docs/docsite/rst/installation_guide/intro_installation.rst

```diff
@@ -293,30 +293,47 @@ Update of the software will be managed by the swupd tool::
 Latest Releases via Pip
 +++++++++++++++++++++++
 
-Ansible can be installed via "pip", the Python package manager.  If 'pip' isn't already available in
-your version of Python, you can get pip by::
+Ansible can be installed via ``pip``, the Python package manager.  If ``pip`` isn't already available on your system of Python, run the following commands to install it::
 
-   $ sudo easy_install pip
+    $ curl https://bootstrap.pypa.io/get-pip.py -o get-pip.py
+    $ python get-pip.py --user
 
-Then install Ansible with [1]_::
+Then install Ansible [1]_::
 
-   $ sudo pip install ansible
+    $ pip install --user ansible
 
 Or if you are looking for the latest development version::
 
-   $ pip install git+https://github.com/ansible/ansible.git@devel
+    $ pip install --user git+https://github.com/ansible/ansible.git@devel
 
-If you are installing on macOS Mavericks, you may encounter some noise from your compiler.  A workaround is to do the following::
+If you are installing on macOS Mavericks (10.9), you may encounter some noise from your compiler. A workaround is to do the following::
 
-   $ sudo CFLAGS=-Qunused-arguments CPPFLAGS=-Qunused-arguments pip install ansible
+    $ CFLAGS=-Qunused-arguments CPPFLAGS=-Qunused-arguments pip install --user ansible
 
-Readers that use virtualenv can also install Ansible under virtualenv, though we'd recommend to not worry about it and just install Ansible globally. Do not use easy_install to install Ansible directly.
+If you need to use the ``paramiko`` connection plugin, install the required module [2]_::
```

Comment:
How would a user know if s/he needs to use the paramiko connection plugin?

Reply:
That's a good question. The only reason I put this in here is because `paramiko` used to get installed with Ansible by default. As of 2.8, it will not be. Most users on macOS have been using `paramiko` as the transport but just did not know. Or if some is used to using `paramiko` as the transport, they will be surprised that it is not longer automatically installed since it has been for a very long time (if not always).

One case where they would need to install `paramiko` is if they are using password authentication with `ssh` connection plugin and are unable to install `sshpass` (it's _slightly_ difficult to install on macOS). `sshpass` is required to use passwords with `ssh`, but isn't required for key-based authentication.

That's a lot of caveats to put there, so I'm not sure how to say all that succinctly. :smile:

Reply:
I guess I can add some more info on what you would want that in the footnote rather than cluttering up the directions.